cmd/juju/controller/destroy_test.go

```diff
@@ -507,7 +507,7 @@ func (s *DestroySuite) TestDestroyCommandConfirmation(c *gc.C) {
 	c.Check(cmdtesting.Stdout(ctx), gc.Matches, "WARNING!.*test1(.|\n)*")
 	checkControllerExistsInStore(c, "test1", s.store)
 
-	for _, answer := range []string{"y", "Y", "yes", "YES"} {
+	for _, answer := range []string{"test1"} {
```

Comment:
Looping over hard-coded slice of length one? Might be worth dropping the loop entirely, unless you think it could be extended in future

Reply:
agreed, we need to drop this cycle